app/controllers/githook_controller.rb

```diff
@@ -3,11 +3,14 @@ class GithookController < ApplicationController
   before_action :authenticate, :only => [:dev, :master]
 
   def index
+    request.body.rewind
+    payload_body = request.body.read
+    verify_signature(payload_body)
     ref = params[:ref]
     success = false
-    if ref == "refs/heads/master"
+    if ref == 'refs/heads/master'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.